developer/deploying/platform.md

```diff
@@ -1,6 +1,6 @@
 # Reaction Platform
 
-The following documentation is for paid users of [Reaction Platform](getrxn.io/reaction-platform).
+The following documentation is for the [Reaction Platform](getrxn.io/reaction-platform).
```

Comment:
Why do we still have this if we are trying to remove references to the Platform?